Fitness/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,221 @@
+        emailField.autocapitalizationType = .none
+        // Padding
+        let paddingView = UIView(frame: CGRect(x: 0, y: 0, width: 15, height: emailField.frame.height))
+        emailField.leftView = paddingView
+        emailField.leftViewMode = UITextField.ViewMode.always
+//        emailField.addTarget(self, action: #selector(textChanged(sender:)), for: .editingDidEndOnExit)
+        view.addSubview(emailField)
+        
+        checkSymbol = UIImageView()
+        checkSymbol.layer.cornerRadius = 12
+        checkSymbol.image = UIImage(named: "green check")
+        checkSymbol.alpha = 0
+        emailField.addSubview(checkSymbol)
+        
+        googleBtn = UIButton()
+        let googleWhite = UIColor(red: 1, green: 1, blue: 1, alpha: 1)
+        let googleBorder = UIColor(red: 0.91, green: 0.91, blue: 0.91, alpha: 1).cgColor
+        googleBtn.backgroundColor = googleWhite
+        googleBtn.layer.borderColor = googleBorder
+        googleBtn.setImage(UIImage(named: "googleLogo"), for: .normal)
+        googleBtn.backgroundColor = googleWhite
+        googleBtn.layer.borderWidth = 1
+        googleBtn.layer.cornerRadius = cornerRadius
+        googleBtn.addTarget(self, action: #selector(googleButtonTapped), for: .touchDown)
+        view.addSubview(googleBtn)
+        
+        nextButton = UIButton()
+        nextButton.clipsToBounds = false
+        nextButton.layer.cornerRadius = nextButtonSize / 2
+        nextButton.backgroundColor = .fitnessWhite
+        nextButton.layer.borderColor = UIColor.fitnessMediumGrey.cgColor
+        nextButton.layer.borderWidth = buttonBorderSize
+        nextButton.addTarget(self, action: #selector(goToNextView), for: .touchDown)
+        nextButton.isEnabled = false
+        nextButtonArrow = UIImageView(image: UIImage(named: "arrow"))
+        nextButtonArrow.transform = CGAffineTransform(rotationAngle: CGFloat(Double.pi))
+        nextButton.addSubview(nextButtonArrow)
+        view.addSubview(nextButton)
+        
+        // If already signed in, make Next Button Yellow
+        toggleNextButton(enabled: isGoogleSignedIn())
+        
+        // Check for google sign in
+        NotificationCenter.default.addObserver(self, selector: #selector(userSignedInWithGoogle), name: NSNotification.Name("SuccessfulSignInNotification"), object: nil)
+        
+        setUpConstraints()
+    }
+    
+    // MARK: UI Control
+    
+    func setUpConstraints() {
+        titleLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(edgePadding)
+            make.leading.trailing.equalToSuperview().inset(edgePadding)
+        }
+        
+        signUpLabel.snp.makeConstraints { (make) in
+            make.top.equalTo(view.safeAreaLayoutGuide).inset(titleToContentPadding)
+            make.leading.trailing.equalToSuperview().inset(edgePadding)
+        }
+        
+//        emailField.snp.makeConstraints { (make) in
```

Comment:
If you don't think we'll use this code (or the previous that initializes it) it in the future, delete it!